### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lgardenhire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,167 +1,87 @@
...
 
-[NVTabular](https://github.com/NVIDIA/NVTabular), a component of [**NVIDIA Merlin Open Beta**](https://developer.nvidia.com/nvidia-merlin), is a feature engineering and preprocessing library for tabular data that is designed to quickly and easily manipulate terabyte scale datasets and train deep learning (DL) based recommender systems. It provides a high level abstraction to simplify code and accelerates computation on the GPU using the [RAPIDS Dask-cuDF](https://github.com/rapidsai/cudf/tree/main/python/dask_cudf) library.
+The RAPIDS Dask-cuDF library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do individual epoch shuffles orders faster than a full shuffle of the datasent. Loading tabular data is almost always a training bottleneck. In our benchmarking, we've seen 10x improvements in training time on the GPU relative to the native dataloaders.
```
Comment (benfred) [inline README.md]: RAPIDS Dask-cuDF library isn't interopable with PT/TF - this is something we're doing in NVT

```suggestion
NVTabular is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do individual epoch shuffles orders faster than a full shuffle of the datasent. Loading tabular data is almost always a training bottleneck. In our benchmarking, we've seen 10x improvements in training time on the GPU relative to the native dataloaders.
```

Comment (benfred) [reply README.md]: I'd like to see a bit more information here about the benefits of our dataloaders - while the shuffling support is cool, I don't think that its the primary advantage of using NVT's dataloaders.

I think instead we should highlight the performance improvements first, and link to the blog post about TF dataloading when talking about the perf improvements.

File: README.md [hunk 2]
```diff
@@ -1,167 +1,87 @@
...
+## Benefits 
 
-NVTabular is designed to support data scientists and machine learning (ML) engineers train (deep learning) recommender systems and resolve tabular data problems by allowing them to:
+Our ultimate goal is faster iteration on massive tabular datasets, both for experimentation during training, and also for production model responsiveness. NVTabular is designed to support data scientists and machine learning (ML) engineers train (deep learning) recommender systems and resolve tabular data problems by allowing them to:
 
-* prepare datasets quickly and easily for experimentation so that more models can be trained
+* prepare datasets quickly and easily for experimentation so that more models can be trained.
 * work with datasets that exceed GPU and CPU memory without having to worry about scale.
 * focus on what to do with the data, and not how to do it, using our abstraction at the operation level.
```
Comment (benfred) [inline README.md]: I think this benefits section should include some extra info:

*  I think one of the core benefits we should highlight is the accelerated dataloaders - we should add a section about quickly training models with Tensorflow, PyTorch.  
* We should mention inference + integration with Triton

File: README.md [hunk 3]
```diff
@@ -1,167 +1,87 @@
...
+## Core Features
 
-Extending beyond model training, we plan on integrating with model-serving frameworks like NVIDIA’s Triton Inference Server. This will create a clear path to production inference for these models and allow the feature engineering and preprocessing steps performed on the data during training to be automatically applied to incoming data during inference.
+NVTabular supports the following core features:
+* [Framework Interoperability](../docs/source/core_features.md#framework-interoperability)
+* [Multi-GPU Support](../docs/source/core_features.md#multi-gpu-support)
+* [Multi-Node Support](../docs/source/core_features.md#multi-node-support)
+* [Multi-Hot Encoding and Pre-existing Embeddings](../docs/source/core_features.md#multi-hot-encoding-and-pre-existing-embeddings)
+* [CPU Support](../docs/source/core_features.md#cpu-support)
```
Comment (benfred) [inline README.md]: I like having this core features list!

Some thoughts on this:
* 'Framework Interoperability' isn't clear what frameworks we're talking about - we should mention something like 'TensorFlow, PyTorch and HugeCTR Interoperability' instead
* CPU Support is currently scheduled for 0.5 - and I don't think we should be highlighting as a core feature yet
* Inference ? Should it be here or in the benefits section?

File: README.md [hunk 4]
```diff
@@ -1,167 +1,87 @@
...
 
-Our goal is faster iteration on massive tabular datasets, both for experimentation during training, and also for production model responsiveness.
+## Installation
 
-### Getting Started
-NVTabular is available in the NVIDIA container repository at the following location: http://ngc.nvidia.com/catalog/containers/nvidia:nvtabular.
+To install NVTabular, ensure that you meet the following prerequisites:
+* CUDA version 10.0+
+* Python version 3.7+
+* Pascal architecture or better (Compute Capability version 6.0)
```
Comment (benfred) [inline README.md]: Maybe we should make it clear that Pascal Architecture refers to the GPU here?

```suggestion
* Pascal architecture or later GPU (Compute Capability version 6.0) 
```

File: README.md [hunk 5]
```diff
@@ -1,167 +1,87 @@
...
+NVTabular is a component of [**NVIDIA Merlin Open Beta**](https://developer.nvidia.com/nvidia-merlin). NVIDIA Merlin is used for building large-scale deep learning recommender systems, which require massive datasets to train, particularly for deep learning based solutions. The transformation of these datasets in order to prepare them for model training can be challenging. Often the time taken to complete the necessary steps, such as feature engineering, categorical encoding, and normalization of continuous variables, exceeds the time it takes to train a model. With NVTabular being a part of the Merlin ecosystem, it also works with the other Merlin components including [HugeCTR](https://github.com/NVIDIA/HugeCTR) and [Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) to provide end-to-end acceleration of recommender systems on the GPU. Extending beyond model training, with NVIDIA’s Triton Inference Server, the feature engineering and preprocessing steps performed on the data during training can be automatically applied to incoming data during inference.
```
Comment (benfred) [inline README.md]: This section reads a little weird: the transition from Merlin, to dataset transformation challenges, and then back to Merlin seems a little muddled to me.

Can we just move the dataset transformation challenges sentences out - or even maybe just delete them?

```suggestion
NVTabular is a component of [**NVIDIA Merlin Open Beta**](https://developer.nvidia.com/nvidia-merlin). NVIDIA Merlin is used for building large-scale deep learning recommender systems, which require massive datasets to train, particularly for deep learning based solutions. With NVTabular being a part of the Merlin ecosystem, it also works with the other Merlin components including [HugeCTR](https://github.com/NVIDIA/HugeCTR) and [Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) to provide end-to-end acceleration of recommender systems on the GPU. Extending beyond model training, with NVIDIA’s Triton Inference Server, the feature engineering and preprocessing steps performed on the data during training can be automatically applied to incoming data during inference.
```
